tests/unit/utils/sparqlhandler/test_sparqlmanager.py

```diff
@@ -240,6 +239,32 @@ def test_select_csvw_catalog_metadata_for_codelist():
     assert result.contact_point == "None"
     assert result.identifier == "Alcohol Content"
 
+def test_select_csvw_dsd_dataset_for_old_style_data_set():
```

Comment:
Can you make it clearer exactly what this test is doing. An `old style` data set only makes sense with the current context we have. It's going to be very difficult to understand what an old style data set is when a developer comes back and tries to understand this in future.